src/utils/signbjstx.js

```diff
@@ -273,6 +275,17 @@ function isBech32(address: string): boolean {
 }
 
 function isScriptHash(address: string, network: bitcoin.Network): boolean {
+
+    // cashaddr hack
+    // Cashaddr format (with prefix) is neither base58 nor bech32, so it would fail
+    // in bitcoinjs-lib-zchash. For this reason, we use legacy format here
+
+    try {
+        if (bchaddr.isCashAddress(address) ) {
```

Comment:
hmm. this is potentially dangerous. Maybe more defensive 
 `
bchaddr.isCashAddress && address.startsWith('bitcoincash:'). 
`

Reply:
Bcash testnet uses 'bchtest:' as prefix, but I am not sure we'll ever support that.